Fabulous.XamarinForms/samples/Calculator/Calculator/Calculator.fs

```diff
@@ -94,9 +94,9 @@ module App =
                 .TextColor(Color.Black) 
 
         View.ContentPage(
-            View.Grid(rowdefs=[ "*"; "*"; "*"; "*"; "*"; "*" ], coldefs=[ "*"; "*"; "*"; "*" ],
+            View.Grid(rowdefs=[ Star; Star; Star; Star; Star; Star ], coldefs=[ Star; Star; Star; Star ],
```

Comment:
Any chance of a more semantically meaningful name than `Star`, perhaps `Fit` or `Proportional` or `Weighted` ...? Just wondering, `Star` might be best I guess

Reply:
While being more semantically meaningful might seem interesting, I think the best for us is to use what people will expect to find.

We're not tied to what Xamarin.Forms/XAML does, and Fabulous.XamarinForms is a good occasion to innovate on this front.

But renaming these names, I'm afraid we'll lose people more than any advantage it might give them.

People (beginners and XAML developers) will search/read docs and stackoverflow, where they will find mostly things on Xamarin.Forms and XAML in general.

So using the same names will be more natural for them.